docs/source/rule-testing.rst

```diff
@@ -73,56 +73,63 @@ For example, to replace a time based field with ``last_hour``::
     ]
   }
 
-  Running Tests
-  ~~~~~~~~~~~~~~~~~~~~~~~
 
-  To run integration tests for the rule processor::
+Running Tests
+~~~~~~~~~~~~~~
 
-    $ python stream_alert_cli.py lambda test --processor rule
+Various tests can be run via the ``stream_alert_cli.py`` script. These tests include the ability to validate rules for
+accuracy and alert outputs for proper configuration.
 
-  To run integration tests for the alert processor::
+When adding new rules, it is only necessary to run tests for the **rule processor**. If making code changes to the alert
+processor, such as adding a new output integration to send alerts to, tests for the **alert processor** should also be performed.
```

Comment:
Spoke to Jack offline about this, but will add a comment here too:

RST doesn't care about line breaks, and considers contiguous blocks of text one entity (ie - a line break followed by another line of text [not am empty line] is considered one flowing string). 

Breaks like this make the RST docs easier to edit in a code editor, without needing to constantly scroll back and forth.